src/graphic/Fast3D/backends/gfx_dxgi.h

```diff
@@ -1,7 +1,5 @@
-#ifndef GFX_DXGI_H
-#define GFX_DXGI_H
 #if defined(ENABLE_DX11) || defined(ENABLE_DX12)
-
+#pragma once
```

Comment:
this is happening after `#if defined(ENABLE_DX11) || defined(ENABLE_DX12)` when the header guards were before it?